src/binding/petsc4py/src/petsc4py/PETSc/Scatter.pyx

```diff
@@ -28,17 +45,82 @@ cdef class Scatter(Object):
 
     #
 
-    def view(self, Viewer viewer=None):
+    def view(self, Viewer viewer=None) -> None:
+        """View the scatter.
+
+        Collective.
+
+        Parameters
+        ----------
+        viewer
+          A `Viewer` instance or `None` for the default viewer.
+
+        See Also
+        --------
+        petsc.VecScatterView
+
+        """
         cdef PetscViewer vwr = NULL
         if viewer is not None: vwr = viewer.vwr
         CHKERR( VecScatterView(self.sct, vwr) )
 
-    def destroy(self):
+    def destroy(self) -> None:
+        """Destroy the scatter.
+
+        Collective.
+
+        See Also
+        --------
+        petsc.VecScatterDestroy
+
+        """
         CHKERR( VecScatterDestroy(&self.sct) )
         return self
 
-    def create(self, Vec vec_from, IS is_from or None,
-               Vec vec_to, IS is_to or None):
+    def create(
+        self,
+        Vec vec_from,
+        IS is_from,
+        Vec vec_to,
+        IS is_to,
```

Comment:
This is one of these rare cases where an argument can be `None`, but there is no default value (or its default value is not `None`). So we have to add typing explicitly.

```suggestion
        Vec vec_from,
        IS is_from: IS | None,
        Vec vec_to,
        IS is_to: IS | None,
```